server/lib/pbench/test/conftest.py

```diff
@@ -63,34 +64,21 @@ def setup(request, pytestconfig):
     cfg_file = pbench_cfg / "pbench-server.cfg"
     cfg_file.write_text(server_cfg_tmpl.format(tmp=tmp_d))
 
-    pbench_recv_tb = (
-        srv_pbench / "pbench-move-results-receive" / "fs-version" / "tarball-002"
-    )
+    tb_path = Path("./server/lib/pbench/test/test-fixtures/upload/log.tar.xz")
```

Comment:
Given that every place you use `tb_path` you either convert it to a string or is in a context where a string is acceptable, I would drop the `Path()` from here and let it just be a string.